### DISCUSSION TITLE
linux quadratic

### DISCUSSION THREAD

**Discussion by duty-prog** [community/community#147889] [Programming Help]
### Body

navigation for code permanency in mx-linux quadratic -open editors yet c++ folder is hope sourialedward09@gmail.com
![Screenshot_20241222_104343](https://github.com/user-attachments/assets/4e436af6-0c9d-4f15-bce8-96ac366aee21)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mecodeatlas** [comment]
Hi @duty-prog, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction :)
